### PR TITLE
Application documentation added.

### PR DESCRIPTION
**Pull request by edwoodward**
Documentation linked in README.

### CODE REVIEW

File: docs/app-design.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+* Contains code for the Blog Atom feed
+* Contains code for blog posts
+* Has APIs for a full list of blog posts or an single blog post
+* Contains code for OS press kit
+* Search is implemented for blog posts
+
+### OpenStax
+* The application used for settings and general items
+* APIs for accessing Documents and Images
+* Document API checks if faculty documents should be hidden or not
```
Comment (mwharrison) [inline docs/app-design.md]: This is one item that uses a feature flag.

File: docs/app-design.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  * Subject
+  * Faculty Resourse
+  * Student Resource
+  * Role
+  * Shared Content
+  * News Source
+* There are APIs for Roles and Subjects
+
+## Other
+* **Import-Export:** Admin has a feature to import or export a page. This uses the Wagtail import-export package. It is used to move pages from one environment to another.
```
Comment (mwharrison) [inline docs/app-design.md]: Here is a link to this project: https://pypi.org/project/wagtail-import-export-tool/

File: docs/app-design.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  * Sent to Customer Support
+  * More Information Requested
+* Errata can be various error types
+  * Other factual inaccuracy in content
+  * General/pedagogical suggestion or question
+  * Incorrect answer, calculation, or solution
+  * Broken link
+  * Typo
+  * Other
+* Emails are sent to users who entered the errata as it moves through the workflow. users can opt out of receiving the emails.
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: I thought users had to give the email address to receive the email. Not giving an email would be opting out. I must be remembering wrong.

File: docs/app-design.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+## CMX Django Application Details
+
+### Accounts
+* Used for accounts in dev and local environments
+* Will not be used for any environment using Cloudfront since Cloudfront redirects /accounts to OS Accounts.
+* The CMS database contains many users because when the app was first deployed, user info was stored in the CMS. That functionality is now in OS Accounts.
+
+### Admin Templates
+* Contains templates for Wagtail login and for adding a redirect URL.
+* Redirect URLs are used in books and are short openstax URLs used for third party sites
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: This has been added

File: docs/app-design.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+### Allies
+* Model for the Partners page
+* It was originally called the Ally page
+* Each Ally has one or more subjects they cover. The list can be filtered by subject in the OS UI.
+* They also fall into 3 possible categories: Online homework, Adaptive courseware and Customization tools
+* The Partner data is pulled from Salesforce. See [Salesforce section](#Salesforce)
+
+### API
+* APIs for various data
+  * images - images used on OS site - is this correct? Cloudfront?
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+  * documents - PDFs, PowerPoints, text files, etc.
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+  * adopters - is this correct? cloudfront?
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison Removed the reference to cloudfront

File: docs/app-design.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+  * progress - what is this?
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison I am wrong. This has been removed.

File: docs/app-design.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+### Extra Admin Filters
+* Used to filter errata
+* Allows users to filter multiple books in the same query
+
+### Global Settings
+* Views and models for items on the Admin Settings menu
+  * Sticky Note
+  * Footer
+  * Cloudfront Distribution
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+* Used to filter errata
+* Allows users to filter multiple books in the same query
+
+### Global Settings
+* Views and models for items on the Admin Settings menu
+  * Sticky Note
+  * Footer
+  * Cloudfront Distribution 
+
+### Mail
```
[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md (hunk 1 above)
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md (hunk 2 above)
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been added

File: docs/app-design.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (mwharrison) [reply docs/app-design.md]: @edwoodward, sorry, I don't think I was clear :)
SVGs live under documents.

[the remark this answers is not in the document]
Comment (edwoodward) [reply docs/app-design.md]: @mwharrison This has been updated

**Pull request merged**